cmake/gtwrapConfig.cmake.in

```diff
@@ -0,0 +1,24 @@
+# This config file modifies CMAKE_MODULE_PATH so that the wrap cmake files may
+# be included This file also allows the use of `find_package(gtwrap)` in CMake.
+
+@PACKAGE_INIT@
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
+
+# Set the path to the Python package directory so we can add it to the PYTHONPATH.
+# Used in the *Wrap.cmake files.
+set_and_check(GTWRAP_PYTHON_PACKAGE_DIR @PACKAGE_INSTALL_LIB_DIR@)
+
+# Load all the CMake scripts from the standard location
+include(@PACKAGE_INSTALL_CMAKE_DIR@/PybindWrap.cmake)
+include(@PACKAGE_INSTALL_CMAKE_DIR@/MatlabWrap.cmake)
```

Comment:
should there be a flag whether or not to install matlab wrap?
On one of my systems, I don't have matlab installed and this throws errors unless I comment-out this line

Reply:
Shouldn't be a flag but we would need to update what happens when we include MatlabWrap.cmake.